src/wp-includes/interactivity-api/class-wp-interactivity-api-directives-processor.php

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
The file docblock, the class docblock, and all method docblocks need `@since 6.5.0` annotations.

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/5953/commits/c0f580454a2ddf4db571f3f5e28f5d3a0822a761.